### ISSUE TITLE
Several questions about the method

### ISSUE THREAD

**Issue by ih4cku** [jocicmarko/ultrasound-nerve-segmentation#3]
Hi, Marko,

Thanks for sharing your codes. I have several questions about the method you use.

**The data** 

Why do use load the mask images as a grayscale image with [0, 255] data instead of converting them to binary mast with [0, 1] as the ground truth which can be interpreted as the probability of being the foreground area. That would be more natural for segmentation.

**The loss** 

Using the official evaluation function as the target function is a bit of indirect for supervising the network. Have you considered formulate the problem as segmentation as use binary cross-entropy loss function? That would be natural to train. In other hand, add a `C` constant lead to another different target function.

Best regards.

**P.S** In case you didn't see this message, I also post on the [Kaggle forum](https://www.kaggle.com/c/ultrasound-nerve-segmentation/forums/t/21358/0-57-deep-learning-keras-tutorial/127451#post127451). If you see it, please reply there.

**Comment by jocicmarko** [comment]
Replied on https://www.kaggle.com/c/ultrasound-nerve-segmentation/forums/t/21358/0-57-deep-learning-keras-tutorial/128747#post128747.
